pkg/pipeline/templates/register.go.tmpl

```diff
@@ -6,7 +6,7 @@
 package apis
 
 import (
-	"k8s.io/apimachinery/pkg/runtime"
+	zz_runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
Would it make sense to use the imports mechanism that prevents such collisions in the first place? https://github.com/crossplane/terrajet/blob/9b7dd2f/pkg/pipeline/setup.go#L56 `zz_runtime` as import alias is a bit unexpected for readers used to Crossplane codebases.